classes/uuu_bootloader_tag.bbclass

```diff
@@ -4,6 +4,7 @@
 do_deploy:append() {
     if [ "${UUU_BOOTLOADER}" != "" ]; then
         cp ${DEPLOYDIR}/${UUU_BOOTLOADER}   ${DEPLOYDIR}/${UUU_BOOTLOADER_TAGGED}
+        cp ${DEPLOYDIR}/${UUU_BOOTLOADER}   ${DEPLOYDIR}/${UUU_BOOTLOADER_UNTAGGED}
         ln -sf ${UUU_BOOTLOADER_TAGGED}     ${DEPLOYDIR}/${UUU_BOOTLOADER}
         stat -L -cUUUBURNXXOEUZX7+A-XY5601QQWWZ%sEND ${DEPLOYDIR}/${UUU_BOOTLOADER} \
```

Comment:
I'd use `UUU_BOOTLOADER_TAGGED` here to make the code more specific and saying.

Reply:
Thanks @MrCry0, it's a valid point, however this was already merged into master and I'm just backporting it to kitkstone now. Let's keep it like this and we can fix it on master and backport the change, if required.

Reply:
@hiagofranco please prepare a PR fixing this as well.

Reply:
Sure, PR opened: https://github.com/Freescale/meta-freescale/pull/1764